PIL/Image.py

```diff
@@ -2142,6 +2237,7 @@ def __transformer(self, box, image, method, data,
         self.im.transform2(box, image.im, method, data, resample, fill)
```

Comment:
Where is this method definition? I'd like to know what `fill` is, why it defaults to a `1`. I don't see where it's used in `_imaging.c/_transform2` either; i.e. I see it's parsed, but I don't see it being used anywhere in that function.